components/splitter/__tests__/index.test.tsx

```
@@ -0,0 +1,352 @@
+import fs from 'fs';
+import path from 'path';
+import React from 'react';
```

ESLint

[error] 'react' imported multiple times.

(import/no-duplicates)